include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -174,20 +174,34 @@ make_iter_mode(const _Iterator& __it) -> decltype(iter_mode<outMode>()(__it))
 }
 
 // set of templated classes to name kernels
+// Shoten the names used in kernel generator to potentially improve compilation time
```

Comment:
Shoten => Shorten
I propose to mention the all unreadable names  defined below in this description.

Reply:
Done. Comments rewritten